net.go

```diff
@@ -0,0 +1,27 @@
+package sqrl
+
+import (
+	"fmt"
+	"log"
+	"os/exec"
+	"strings"
+
+	"github.com/shirou/gopsutil/net"
+)
+
+func netInfo() {
+	a, _ := net.Interfaces()
+
+	fmt.Println()
+	fmt.Println("\t\t\tNetwork Info")
+
+	netSpeed, err := exec.Command("ethtool", "eno16777984").Output()
+	if err != nil {
+		log.Fatal(err)
+	}
+	output1 := string(netSpeed)
+	result := strings.Split(output1, "\n")
+
+	fmt.Println(a[1].Name, a[1].Addrs, a[1].Flags, result[9])
```

Comment:
two things here...

1) what you have named "a" is a []InterfaceStat, which means it has multiple values.  As it stands right now, you are only investigating the second element in the slice.  Make sure you get all of them.  Remember, a machine can have multiple interfaces on them!

2)
I totally understand what you are trying to do here and for the first shot, it is a great attempt to just print the values out. Let me give you some ideas of how I want you to move forward.

Lets go ahead and define a struct that encompasses all the data you need to get.

```
type Interfaces struct {
    interface IPAddr, Mac string
    speed int
    link string
}
```
Or something like this struct I just defined.... 

Create me a function that I can call and it will return me a Interfaces struct.

Hopefully that makes sense...